src/components/modal/styles.ts

```diff
@@ -23,40 +29,49 @@ export const ElModal = styled.div`
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  box-shadow: 0px 4px 16px 0px rgba(34, 43, 51, 0.16);
-  border-radius: var(--default-border-radius);
   background: white;
   z-index: 99;
-  width: 65%;
-  min-width: 300px;
-  max-width: 800px;
-  max-height: 80vh;
   overflow: auto;
-  padding: 1.25rem 1.5rem;
+  box-shadow: var(--elevation-xl, 0px 4px 16px 0px rgba(96, 120, 144, 0.16));
+  border-radius: var(--corner-lg, 8px);
+  padding: var(--spacing-5, 20px) var(--spacing-6, 24px);
```

Comment:
talking about the transition when opening and closing the dialog, so I make a query to Design team

reference https://www.figma.com/design/J6EneNGfpk2GcUuymiOKW4?node-id=164-4207&m=dev#978865056

Reply:
> The background scrim fading in/out? If it helps, please just build the component exactly the same as Core UI. Thank you!

From Karmen as per conversation in Figma above, copying the fading transition like the Core UI did 
 in the next commit https://app.console.com.au/client-apps/design-system/react-ui-core/?path=/docs/overlay-dialog--docs

Reply:
after looking a while, looks more better if I add the fade in/fade out transition on part 2 which it would also use the new `dialog` html API and restructuring the component with the new pattern

Reply:
> ...part 2 which it would also use the new `dialog` html API and restructuring the component with the new pattern

**note:** If we are going to move to the native `<dialog>` element, I think it would have been even more valuable to leave this Modal component as-is (and deprecate it) and just implement a brand new Dialog component (as I noted earlier in this PR).

**note:** Also, it's helpful to describe any staging of changes you are planning in the PR description. It can save reviewers a lot of time/questions if you outline when certain expected changes will be done.